libs/ui/src/hooks/smartcard_auth/auth_helpers.test.ts

```diff
@@ -30,6 +37,18 @@ const allowedUserRoles: UserRole[] = [
   'cardless_voter',
 ];
 
+function assertAndGetPostDipCard(
+  result: RenderResult<DippedSmartcardAuth.Auth>
+): CardStorage & CardProgramming {
+  assert(isSystemAdministratorAuth(result.current));
+  const { card } = result.current;
+  expect(card).not.toBe('no_card');
```

Comment:
I've started to always use `.toEqual` instead of a mix of `.toEqual` and `.toBe`. Though my opinions aren't as strong as the [title of this article](https://dev.to/thejaredwilcurt/why-you-should-never-use-tobe-in-jest-48ca), it raises some good points about the benefits of always using `.toEqual`!

Reply:
Ooh good call, I am down to adopt that convention. Wonder if there's an easy lint rule for it

Reply:
I don't think [eslint-plugin-jest](https://github.com/jest-community/eslint-plugin-jest#readme) has that, but we could write our own.

Reply:
Nice, glad we're in alignment! Made an issue to track: https://github.com/votingworks/vxsuite/issues/2510